lib/ansible/modules/cloud/amazon/rds_instance.py

```diff
@@ -0,0 +1,1267 @@
+            should be applied as soon as possible, regardless of the I(preferred_maintenance_window) setting. If false, changes
+            are applied during the next maintenance window.
+        type: bool
+        default: False
+    auto_minor_version_upgrade:
+        description:
+          - Whether minor version upgrades are applied automatically to the DB instance during the maintenance window.
+        type: bool
+    availability_zone:
+        description:
+          - A list of EC2 Availability Zones that instances in the DB cluster can be created in.
+            May be used when creating a cluster or when restoring from S3 or a snapshot. Mutually exclusive with I(multi_az).
+        aliases:
+          - az
+          - zone
+    backup_retention_period:
+        description:
+          - The number of days for which automated backups are retained (must be greater or equal to 1).
+            May be used when creating a new cluster, when restoring from S3, or when modifying a cluster.
+    ca_certificate_identifier:
+        description:
+          - The identifier of the CA certificate for the DB instance.
+    character_set_name:
+        description:
+          - The character set to associate with the DB cluster.
+    copy_tags_to_snapshot:
+        description:
+          - Whether or not to copy all tags from the DB instance to snapshots of the instance. When initially creating
+            a DB instance the RDS API defaults this to false if unspecified.
+        type: bool
+    db_cluster_identifier:
+        description:
+          - The DB cluster (lowercase) identifier to add the aurora DB instance to. The identifier must contain from 1 to
+            63 letters, numbers, or hyphens and the first character must be a letter and may not end in a hyphen or
+            contain consecutive hyphens.
+        aliases:
+          - cluster_id
+    db_instance_class:
+        description:
+          - The compute and memory capacity of the DB instance, for example db.t2.micro.
+        aliases:
+          - class
+          - instance_type
+    db_instance_identifier:
+        description:
+          - The DB instance (lowercase) identifier. The identifier must contain from 1 to 63 letters, numbers, or
+            hyphens and the first character must be a letter and may not end in a hyphen or contain consecutive hyphens.
+        aliases:
+          - instance_id
+          - id
+        required: True
+    db_name:
+        description:
+          - The name for your database. If a name is not provided Amazon RDS will not create a database.
+    db_parameter_group_name:
+        description:
+          - The name of the DB parameter group to associate with this DB instance. When creating the DB instance if this
+            argument is omitted the default DBParameterGroup for the specified engine is used.
+    db_security_groups:
```

Comment:
@dmsimard To make this more like ec2_instance I was going to consolidate `db_security_groups` and `vpc_security_group_ids` into one `security_groups` option (retaining the original option names as aliases), but decided not to since I don't have access to an EC2-Classic account and cannot test the edge cases.

Reply:
Yeah I think it's fair to keep the module consistent with the API (despite being inconsistent with other AWS modules, it's not technically out fault) for the sake of keeping things maintainable. It's a shame from a UX perspective though.